multiview/construct/rsm.py

```diff
@@ -0,0 +1,125 @@
+"""
+rsm.py
+====================================
+Random subspace method for view construction.
+"""
+
+import numpy as np
+import random
+
+
+def random_subspace_method(X, n_features=None, n_views=1):
+    """
+    Random Subspace Method for constructing multiple views.
+    Each view is constructed by randomly selecting n_features
+    (columns) from X. All unselected features are set to 0.
+
+    Original paper: https://ieeexplore.ieee.org/document/709601
+
+    Parameters
+    ----------
+    X : array-like matrix, shape = [n_rows, n_cols]
+        The input samples.
+
+    n_features : int, float
+        Number of features to randomly select.
+
+        - If int, then consider n_features as number of columns
+        to select.
+
+        - If float, then consider n_features*n_cols as number of columns
+        to select.
+
+    n_views : int, float optional (default = 1)
+        Number of views to construct.
+
+    Returns
+    -------
+    views : list of array-like matrices
+        List of constructed views (each matrix has shape [n_rows, n_cols]).
+
+    """
+
+    # todo let max_features be float for probability
```

Comment:
Merge fine if this todo is a future todo

Reply:
Ooh good catch. Already implemented just forgot to delete comment.